data/859/026/35/85902635.geojson

```diff
@@ -44,6 +44,9 @@
     "name:fin_x_variant":[
         "K\u00e1to \u00cdlia"
     ],
+    "name:gre_x_preferred_x_variant":[
```

Comment:
Same property name problems as noted elsewhere.

Reply:
This should be fixed in the most recent commit.